model/job_test.go

```diff
@@ -222,3 +223,36 @@ func TestJobsSort(t *testing.T) {
 	assert.Equal(jobs[0].Name, "ccc")
 	assert.Equal(jobs[1].Name, "ddd")
 }
+
+func TestJobsProperties(t *testing.T) {
+	assert := assert.New(t)
+
+	workDir, err := os.Getwd()
+	assert.NoError(err)
+
+	ntpReleasePath := filepath.Join(workDir, "../test-assets/ntp-release")
+	ntpReleasePathBoshCache := filepath.Join(ntpReleasePath, "bosh-cache")
+	release, err := NewDevRelease(ntpReleasePath, "", "", ntpReleasePathBoshCache)
+	assert.NoError(err)
+
+	assert.Equal(1, len(release.Jobs))
```

Comment:
Nope, the [docs](https://godoc.org/github.com/stretchr/testify/assert#Assertions.Len) say it's the thing first, and length second.  I guess they think consistency is overrated?